include/swift/Driver/Job.h

```diff
@@ -271,6 +271,14 @@ class Job {
   /// Whether the job wants a list of input or output files created.
   std::vector<FilelistInfo> FilelistFileInfos;
 
+  /// Response file path
```

Comment:
What's the point of this comment? @jrose-apple Is there some coding standard I don't know about? Why not remove it entirely?

Reply:
This purpose of the comment is to remain consistent with the rest of the file. A few lines above reads:
```
/// The output of this command.
std::unique_ptr<CommandOutput> Output;

/// The executable to run.
const char *Executable;
```
for example.

Reply:
I understand. Since I believe that this sort of comment degrades maintainability (yes, it's a tiny cut but there are virtually thousands of them), and since I try to enhance the maintainabilty of the code with each review, I asked for its deletion. However, a consistent-but-redundant comment is only a mere bagatelle.